src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -12,6 +12,7 @@
   --#{$breadcrumb}__item-divider--FontSize: var( --pf-t--global--font--size--body--sm);
 
   // Breadcrumb link
+  --#{$breadcrumb}__link--PaddingInlineStart: revert; // use a mutable value for alignment control
```

Comment:
Were changed to this file meant to be in this PR?

Reply:
Looks like this is causing the focus styles to look like this:

![image](https://github.com/patternfly/patternfly/assets/70952936/fc9b40fd-0bca-47f2-b3cb-b361ec23a53b)

Disabling this inline start padding causes the link text to be more centered horizontally in the focus ring